msteams-platform/concepts/deploy-and-publish/apps-upload.md

```diff
@@ -4,120 +4,96 @@ description: Describes how to upload your app in Microsoft Teams
 ms.topic: how-to
 keywords: teams apps Upload
 ---
-# Upload an app package to Microsoft Teams
-
-To test your app experience within Microsoft Teams, you need to upload your app to Teams. Uploading adds the app to the team you select, and you and your team members can interact with it like end users.
 
-> [!NOTE]
-> Uploading an updated package for an existing app with a bot might not show tab changes when viewed through the Conversations window. It's better to access it via the Apps fly-out, or test on a clean test environment.
-
-## Create your upload package
+# Upload an app package to Microsoft Teams
+This document guides you on the process to upload your apps to your organization environment through different methods.
 
-For development as well as AppSource (formerly Office Store) submission you must create an uploadable package that contains the information to describe your experience. The package, a .zip file, contains the application manifest and icons that uniquely define your experience.
+After you [create an app package](../build-and-test/apps-package.md), upload your app package and test the app experience within Microsoft Teams. After uploading, the package is available to all users of the selected team, and only users of that team.
 
-To create an upload package, see [Create the package for your Microsoft Teams app](../build-and-test/apps-package.md).
+Before uploading the app package, ensure that the [prerequisite](#prerequisite) is fulfilled. Now, use one of the following methods to upload your apps to Teams:
+* [Upload using the App Store](#upload-your-package-into-a-team-or-conversation-using-the-store)
+* [Upload using the Apps tab](#upload-your-package-into-a-team-using-the-apps-tab)
```

Comment:
```suggestion
```
I think we can remove this method and any related steps. The main use case is through the store.